pkg/didcomm/protocol/didexchange/service_test.go

```diff
@@ -530,7 +527,7 @@ type mockStore struct {
 	get func(string) ([]byte, error)
 }
 
-// Put stores the key and the record
+// Store stores the key and the record
```

Comment:
The comment was updated but not the method name?

Reply:
done